### PR TITLE
moving the creation of the sumbolic link between IDRM.h into cdmi.h i…

### PR DESCRIPTION
**Pull request by wiswis15**
moving the creation of the sumbolic link between IDRM.h into cdmi.h into the interface project

### CODE REVIEW

File: package/wpe/wpeframework-interfaces/wpeframework-interfaces.mk [hunk 1]
```diff
@@ -34,4 +34,8 @@ else ifeq ($(BR2_PACKAGE_WPEFRAMEWORK_BUILD_TYPE_PRODUCTION),y)
         WPEFRAMEWORK_INTERFACES_CONF_OPTS += -DBUILD_TYPE=Production
 endif
 
+ifeq ($(BR2_PACKAGE_WPEFRAMEWORK_CDM),y)
```
Comment (artur-gebicz-ml) [inline package/wpe/wpeframework-interfaces/wpeframework-interfaces.mk:37]: Is this change related? If yes, then why it's needed now and not before?

File: package/wpe/wpeframework/wpeframework.mk [hunk 2]
```diff
@@ -183,14 +183,6 @@ define WPEFRAMEWORK_POST_TARGET_REMOVE_HEADERS
 	rm -rf $(TARGET_DIR)/usr/include/WPEFramework
 endef
 
-define WPEFRAMEWORK_POST_STAGING_CDM_HEADER
-	ln -sfn $(STAGING_DIR)/usr/include/WPEFramework/interfaces/IDRM.h $(STAGING_DIR)/usr/include/cdmi.h
```
Comment (artur-gebicz-ml) [inline package/wpe/wpeframework/wpeframework.mk:187]: This will break the backward compatibility. There still might be a code that does not include IDRM.h and use cdmi.h instead. IMHO this should be moved to wpeframework-interfaces

**Review by artur-gebicz-ml — CHANGES_REQUESTED** [review]

File: package/wpe/wpeframework-interfaces/wpeframework-interfaces.mk (hunk 1 above)
Comment (wiswis15) [reply package/wpe/wpeframework-interfaces/wpeframework-interfaces.mk:37]: it was also done before

File: package/wpe/wpeframework/wpeframework.mk (hunk 2 above)
Comment (artur-gebicz-ml) [reply package/wpe/wpeframework/wpeframework.mk:187]: https://github.com/rdkcentral/ThunderInterfaces/pull/64/files

**Review by artur-gebicz-ml — APPROVED** [review]
